src/style.css

```
@@ -0,0 +1,120 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --color-dark: #060b35;
+  --color-grey: #616070;
+  --color-blue: #00acdc;
+  --color-white: #fff;
+}
+    
+body {
+  margin: 0;
+  padding: 0;
+}
+    
+.container {
+  width: 198px;
```

```suggestion
  width: 200px;
```
add * {box-sizing: border-box}